testsuite/integration/manualmode/src/test/java/org/wildfly/test/manual/microprofile/health/MicroProfileHealthDefaultEmptyStartupHTTPEndpointTestCase.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2021, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.test.manual.microprofile.health;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.jboss.as.arquillian.container.ManagementClient;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonReader;
+import javax.json.JsonValue;
+import java.io.IOException;
+import java.io.StringReader;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * @author <a href="http://xstefank.io/">Martin Stefanko</a> (c) 2021 Red Hat inc.
+ */
+public class MicroProfileHealthDefaultEmptyStartupHTTPEndpointTestCase extends MicroProfileHealthDefaultEmptyStartupTestBase {
+
+    void checkGlobalOutcome(ManagementClient managementClient, String operation, boolean mustBeUP, String probeName) throws IOException {
+
+        assertEquals("check-started", operation);
+        final String httpEndpoint = "/health/started";
+
+        final String healthURL = "http://" + managementClient.getMgmtAddress() + ":" + managementClient.getMgmtPort() + httpEndpoint;
+
+        try (CloseableHttpClient client = HttpClients.createDefault()) {
+
+            CloseableHttpResponse resp = client.execute(new HttpGet(healthURL));
```

Comment:
Could `try-with-resources` or `try-finally` be used here for `resp` to be sure it gets closed?

Reply:
Thanks, see you've addressed my comment here. This thread can be resolved.